src/test/java/edu/ksu/canvas/attendance/services/AttendanceSummaryCSVServiceUTest.java

```diff
@@ -39,8 +41,10 @@
     private final int ARBITRARY_STUDENT_TOT_MADEUP_2 = 0;
     private final int ARBITRARY_STUDENT_TOBE_MADEUP_2 = 6;
     private final int ARBITRARY_STUDENT_EXCUSED_2 = 1;
+    private final int ARBITRARY_STUDENT_PRESENT_2 = 3;
     private final int ARBITRARY_STUDENT_MIN_MISSED_2 = 100;
     private final double ARBITRARY_STUDENT_PERCENT_2 = 50.1;
+    private final String ARBITRARY_SIS_USER_ID_2 = "SIS_ID_2";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1170)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "ARBITRARY_SIS_USER_ID_2" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00116)